packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.58",
+  "version": "5.0.0",
```

Comment:
This seems unintended 🤔

Reply:
Indeed. Fixed, also fixed the @mui/lab package version

Reply:
I think we can keep `5.0.0-beta.58`, no need to bump it

Reply:
> I always have doubts when `pnpm release:version` suggests bumping the version.

True, we could filter it so it's not prompted

> Do you think we can also keep @mui/lab the same and not bump it?

Yes, if there are no changes I think we can skip it as well

Reply:
ok, changes undone to both packages